src/kani-compiler/src/codegen_cprover_gotoc/codegen/statement.rs

```diff
@@ -497,7 +507,9 @@ impl<'tcx> GotocCtx<'tcx> {
         if self.place_ty(p).is_unit() {
             e.as_stmt(Location::none())
         } else {
-            self.codegen_place(&p).goto_expr.assign(e, Location::none())
+            unwrap_or_codegen_unimplemented_stmt!(self, self.codegen_place(&p))
```

Comment:
Is there a way to avoid repeating `self`, `self.codegen_place`

Reply:
I had the same thought, but couldn't figure out a way to do it. I initially tried not passing `self` in, and just calling it from inside the macro, but that didn't work due to macro hygiene: https://stackoverflow.com/a/44124477.